src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -90,6 +92,12 @@ public ProAI(final String name, final String type) {
     storedPurchaseTerritories = null;
     storedPoliticalActions = null;
     storedStrafingTerritories = new ArrayList<Territory>();
+    initializeBattleCalculator();
+  }
+
+  public void initializeBattleCalculator() {
```

Comment:
protected/public is roughly the same, at the end of the day it is some other class that is telling this class which data state it should be in. Meaning those other classes are aware of what that data state is and should be, this is tight coupling and violation of encapsulation. IOC/dependency injection is an appropriate way to remove that coupling.